crates/bevy_input/src/axis.rs

```diff
@@ -32,4 +42,90 @@ where
     pub fn remove(&mut self, axis: T) -> Option<f32> {
         self.axis_data.remove(&axis)
     }
+
+    pub fn get_min(&self) -> f32 {
+        self.min
+    }
+
+    pub fn get_max(&self) -> f32 {
+        self.max
+    }
+}
+
+const AXIS_MIN: f32 = -1.0;
```

Comment:
These need to be configurable in some way if they're going to be useful.

I would normally have these be an associated constant for a trait bound on `T`, but since T is unconstrained that's not immediately an option 🤔

Reply:
What use-cases are there for changing the axis range? Wouldn't multiplying, offsetting, etc. normally happen on the consuming end?